localization/ndt_scan_matcher/include/ndt_scan_matcher/hyper_parameters.hpp

```diff
@@ -29,6 +29,12 @@ enum class ConvergedParamType {
   NEAREST_VOXEL_TRANSFORMATION_LIKELIHOOD = 1
 };
 
+enum class CovarianceEstimationType {
+  LAPLACE_APPROXIMATION = 0,
+  MULTI_NDT = 1,
+  MULTI_NDT_SCORE = 2,
+};
```

Comment:
How about adding the traditional "set fixed values" mode here?
In that case, I think the `CovarianceEstimation.enabled` option would be unnecessary.

Reply:
I agree with your opinion. 
I'd like to hear other people's opinions as well.

Reply:
@KYabuuchi 
What do you think about this?

Reply:
I also think it seems fine to add a fixed value mode and remove `enabled` from the parameter. :ok_hand:

Reply:
Thank you for your comment.
I will add a fixed value mode and remove `enabled` from the parameters.